homeassistant/helpers/json.py

```diff
@@ -13,13 +13,39 @@
 
 from homeassistant.util.file import write_utf8_file, write_utf8_file_atomic
 from homeassistant.util.json import (  # noqa: F401
-    JSON_DECODE_EXCEPTIONS,
-    JSON_ENCODE_EXCEPTIONS,
+    JSON_DECODE_EXCEPTIONS as _JSON_DECODE_EXCEPTIONS,
+    JSON_ENCODE_EXCEPTIONS as _JSON_ENCODE_EXCEPTIONS,
     SerializationError,
     format_unserializable_data,
-    json_loads,
+    json_loads as _json_loads,
 )
 
+from .deprecation import (
+    DeprecatedConstant,
+    all_with_deprecated_constants,
+    check_if_deprecated_constant,
+    deprecated_function,
+    dir_with_deprecated_constants,
+)
+
+_DEPRECATED_JSON_DECODE_EXCEPTIONS = DeprecatedConstant(
+    _JSON_DECODE_EXCEPTIONS, "homeassistant.util.json.JSON_DECODE_EXCEPTIONS", "2025.2"
```

Comment:
```suggestion
    _JSON_DECODE_EXCEPTIONS, "homeassistant.util.json.JSON_DECODE_EXCEPTIONS", "2025.8"
```

The deprecation period of constants (or anything which can't be fixed by the end-user) is 1 year